pkg/cmd/helmfile/move/move_test.go

```diff
@@ -1,12 +1,12 @@
-package move_test
+package move
```

Comment:
why this change? We want to only test exported functions.

Reply:
Fixed.